### PR TITLE
Allow setting of user ssh keys on GCP, this is needed for nightly CI.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

**Comment by mikalstill** [comment]
PR merge test failures look like a slow CI node (instance did not trigger within timeout).

File: deploy/ansible/deployandtest.sh [hunk 1]
```diff
@@ -52,6 +52,12 @@ then
     exit 1
   fi
   VARIABLES="$VARIABLES project=$GCP_PROJECT"
+
+  if [ -n "$GCP_SSH_KEY" ]
```
Comment (mandoonandy) [inline deploy/ansible/deployandtest.sh:56]: Probably good to name it `GCP_SSH_KEY_FILENAME` to be clear?

File: docs/installation.md [hunk 2]
```diff
@@ -219,6 +219,8 @@ with real users.
 | AWS_VPC_ID | aws, aws-single-node | The AWS VPC to use |
 | AWS_SSH_KEY_NAME | aws, aws-single-node | The name of an SSH key in the AWS region to use for ansible |
 | GCP_PROJECT | gcp | The GCP project id to deploy in |
+| GCP_SSH_KEY | gcp | The content of an ssh public key file to use for authentication (optional, only required if not using gcloud). |
```
Comment (mandoonandy) [inline docs/installation.md:222]: This is used as a filename not the "content"?

Sorry for being retentive.

**Review by mandoonandy — CHANGES_REQUESTED** [review]

File: deploy/ansible/deployandtest.sh (hunk 1 above)
Comment (mikalstill) [reply deploy/ansible/deployandtest.sh:56]: Done.

File: docs/installation.md (hunk 2 above)
Comment (mikalstill) [reply docs/installation.md:222]: Done.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**